src/measurementconfig.cpp

```diff
@@ -17,6 +17,11 @@ MeasurementConfig::MeasurementConfig() : isSaved(false){
     this->distanceInterval = 0.0;
     this->typeOfReading = ePolarReading;
 
+    this->isStablePoint = false;
+    this->stablePointMinDistance = 10.0;
+    this->stablePointThresholdRange = 0.1;
+    this->stablePointThresholdTime = 2.0;
```

Comment:
hat jmd. diese Werte vorgegeben?
wenn die Messung erst nach den Zwei Sekunden beginnt, ist der Zeitraum schon recht lang.
1.0 - 1.5 Sekunden sollten hier ausreichen

Reply:
Ist alles in der Oberfläche konfigurierbar, ich schau mal, wie es sich in der Realität anfühlt.